srcCs/perftest_publisher.cs

```diff
@@ -1267,6 +1257,118 @@ bool ParseConfig(string[] argv)
             return true;
         }
 
+        private void PrintConfiguration() {
+
+            StringBuilder sb = new StringBuilder();
+
+            sb.Append("\nPerftest Configuration:\n");
+
+            // Throughput/Latency mode
+            if (_IsPub) {
+                sb.Append("\tMode: ");
+                if (_LatencyTest) {
+                    sb.Append("Latency (Ping-Pong test)\n");
+                } else {
+                    sb.Append("Throughput (Use \"-latencyTest\" for Latency Mode)\n");
+                }
+
+                sb.Append("\tLatency count: 1 latency sample every ");
+                sb.Append(_LatencyCount);
+                sb.Append("\n");
+            }
+
+            // Reliable/Best Effort
+            sb.Append("\tReliability: ");
+            if (_isReliable) {
+                sb.Append("Reliable\n");
+            } else {
+                sb.Append("Best Effort\n");
+            }
+
+            // Keyed/Unkeyed
+            sb.Append("\tKeyed: ");
+            if (_isKeyed) {
+                sb.Append("Yes\n");
+            } else {
+                sb.Append("No\n");
+            }
+
+            // Publisher/Subscriber and Entity ID
+            if (_IsPub) {
+                sb.Append("\tPublisher ID: ");
+                sb.Append(_PubID);
+                sb.Append("\n");
+            } else {
+                sb.Append("\tSubscriber ID: ");
+                sb.Append(_SubID);
+                sb.Append("\n");
+            }
+
+            // Scan/Data Sizes
+            sb.Append("\tData Size: ");
+            if (_isScan) {
+                for (int i = 0; i < _scanDataLenSizes.Count; i++ ) {
+                    sb.Append(_scanDataLenSizes[i]);
+                    if (i == _scanDataLenSizes.Count - 1) {
+                        sb.Append("\n");
+                    } else {
+                        sb.Append(", ");
+                    }
+                }
+            } else {
+                sb.Append(_DataLen);
+                sb.Append("\n");
+            }
+
+            // Batching
+            sb.Append("\tBatching: ");
+            if (_BatchSize != 0) {
+                sb.Append(_BatchSize);
+                sb.Append(" Bytes (Use \"-batchSize 0\" to dissable batching)\n");
```

Comment:
remove space between " and B
   
    sb.Append("Bytes (Use \"-batchSize 0\" to dissable batching)\n");

Reply:
I will actually do the oposite, I will add the spaces for the other cases, since I believe this:

```
Batching: 2003 Bytes (Use "-batchSize 0" to disable batching
```

Looks nicer than this:

```
Batching: 2003Bytes (Use "-batchSize 0" to disable batching
```

Reply:
Great.